server/src/main/java/org/elasticsearch/index/mapper/vectors/VectorEncoderDecoder.java

```diff
@@ -84,6 +84,26 @@ public static void decodeDenseVector(IndexVersion indexVersion, BytesRef vectorB
         }
     }
 
+    /**
+     * Decodes a BytesRef into the provided array of bytes
+     * @param vectorBR - dense vector encoded in BytesRef
+     * @param vector - array of bytes where the decoded vector should be stored
+     */
+    public static void decodeDenseVector(IndexVersion indexVersion, BytesRef vectorBR, byte[] vector) {
```

Comment:
Needed a specific method for decoding dense vector of byte values - this is an adaptation of the existing float[] method (expand up to see it)